### PR TITLE
Dialogs workspace updates

### PR DESCRIPTION
**Pull request by kisantia**
- Make commands more generic and not project specific because we're adding open workspace as an option
- Add option to open workspace to the open dialog (still waiting on icons from UX)
Screen Shot 2020-10-20 at 4 47 23 PM
- Add workspace that will be created/the project will be added to the dialog so the user knows
Screen Shot 2020-10-20 at 4 48 21 PM
Screen Shot 2020-10-20 at 4 49 10 PM

### CODE REVIEW

File: extensions/data-workspace/src/common/constants.ts [hunk 1]
```diff
@@ -26,18 +27,24 @@ export const DefaultButtonWidth = '80px';
...
 export const NewProjectDialogTitle = localize('dataworkspace.NewProjectDialogTitle', "Create new project");
-export const ProjectTypeSelectorTitle = localize('dataworkspace.ProjectTypeSelectorTitle', "Type");
+export const TypeTitle = localize('dataworkspace.Type', "Type");
 export const ProjectNameTitle = localize('dataworkspace.projectNameTitle', "Name");
 export const ProjectNamePlaceholder = localize('dataworkspace.projectNamePlaceholder', "Enter project name");
 export const ProjectLocationTitle = localize('dataworkspace.projectLocationTitle', "Location");
-export const ProjectLoacationPlaceholder = localize('dataworkspace.projectLocationPlaceholder', "Select folder");
+export const ProjectLocationPlaceholder = localize('dataworkspace.projectLocationPlaceholder', "Enter project location");
+export const AddProjectToCurrentWorkspace = localize('dataworkspace.AddProjectToCurrentWorkspace', "This project will be added to the currently open workspace.");
+export const NewWorkspaceWillBeCreated = localize('dataworkspace.NewWorkspaceWillBeCreated', "New empty workspace will be created for this project.");
```
Comment (alanrenmsft) [inline extensions/data-workspace/src/common/constants.ts:36]: how about removing 'Empty'? : New workspace will be created for this project?

**Review by alanrenmsft — APPROVED** [review]

**Comment by alanrenmsft** [comment]
the commit history is a bit messed up, did you reuse an old branch?

**Review by Charles-Gagnon — APPROVED** [review]

**Comment by kisantia** [comment]
@alanrenmsft yeah I originally created this branch off of the createWorkspace branch before it was checked in. I forgot to rebase after that got checked in :/

**Review by Benjin — APPROVED** [review]

**Comment by kisantia** [comment]
added the string changes to https://github.com/microsoft/azuredatastudio/pull/13022 since this PR already got merged

**Pull request merged**